src/utils/detect-server-settings.mjs

```diff
@@ -300,6 +370,19 @@ const detectServerSettings = async (devConfig, options, projectDir) => {
 
     const runDetection = !hasCommandAndTargetPort({ devConfig })
     const frameworkSettings = runDetection ? await detectFrameworkSettings({ projectDir }) : undefined
+    const newSettings = runDetection ? await detectSettings(projectDir) : undefined
+
+    // just report differences in the settings
+    detectChangesInNewSettings(frameworkSettings, newSettings || [], {
+      ...metadata,
+      settings: {
+        projectDir,
+        devConfig,
+        options,
+        old: frameworkSettings,
+        settings: newSettings,
```

Comment:
Thoughts on calling settings `new` here as well like you did in `detectChangesInNewSettings`? 

```
...
old: frameworkSettings
new: newSettings
...
```

Reply:
it's just debugging info in bugsnag

Reply:
Cool, although I think the old/new contrast makes it easy to understand the intent

Reply:
it's just for me ^^